locales/base/translation.json

```diff
@@ -826,6 +826,7 @@
     "createNewWallet": "Create new wallet",
     "restoreWallet": "Restore wallet",
     "hasWallet": "I already have a wallet",
+    "hasWalletv2": "I have a wallet",
```

Comment:
any reason to use a new variant here? If we need one, should follow the pattern of how other strings are versioned. E.g., `V1_xx`. there should be other examples in this file

Reply:
I was just following what was in the designs, should I update `hasWallet` to remove the already, or just use hasWallet as is?